addons/website/views/website_templates.xml

```diff
@@ -836,24 +854,24 @@
                             </small>
                         </div>
                         <div class="col-lg-4 text-lg-right">
-                            <div class="s_share no_icon_color" data-snippet="s_share" data-name="Social Media">
+                            <div class="s_share s_social_media no_icon_color" data-snippet="s_social_media" data-name="Social Media">
                                 <small class="s_share_title text-muted d-none"><b>Follow us</b></small>
-                                <a href="/website/social/facebook" class="s_share_facebook" target="_blank">
+                                <a data-social-id="social_facebook" href="/website/social/facebook" class="s_share_facebook o_not_editable" target="_blank">
```

Comment:
What is the o_not_editable used for in this case?

Reply:
If I don't set it, there is a vertical bar in the logo and the user can type text.